include/mbgl/style/layer.hpp

```diff
@@ -88,6 +89,8 @@ class Layer : public mbgl::util::noncopyable {
             return std::forward<V>(visitor)(*as<CustomLayer>());
         case LayerType::FillExtrusion:
             return std::forward<V>(visitor)(*as<FillExtrusionLayer>());
+        default:
```

Comment:
Adding `default` means that clang won't issue an "enumeration value not handled in switch" error if you add a new layer type enum but forget to add it here. Use [this convention](https://github.com/mapbox/mapbox-gl-native/blob/c80f3e9d29d1c26ccc88ef30f8f17329c9bfb1b7/src/mbgl/renderer/render_source.cpp#L28-L30) instead.